### DISCUSSION TITLE
Unable to obtain new data modified using tauri cli

### DISCUSSION THREAD

**Discussion by RSS1102** [tauri-apps/tauri#10446] [Q&A]
I use **Tauri's** ` use std:: process:: Command` I tried to change the registry of npm successfully, but the registry obtained is still the one before starting **Tauri**
1. starting **Tauri** (The current npm registry is https://registry.npmjs.org)；
2. set
```rs
    let _= Command::new("cmd")
        .args(&["/C", "npm config set registry http://mirrors.cloud.tencent.com/npm"])
        .output()
        .expect("failed");
```
3.get **(At this moment, what I have obtained is https://registry.npmjs.org)**
```rs
    let output = Command::new("cmd")
        .args(&["/C", "npm config get registry"])
        .output()
        .expect("failed");

    let version = String::from_utf8_lossy(&output.stdout);
    return version.to_string();
```
But after I closed **Tauri ** and ran 'npm config get registry' again, what I got was http://mirrors.cloud.tencent.com/npm

So was this environment variable set by npm cached when ** auri** started?

**Comment by FabianLars** [comment]
Moved to https://github.com/tauri-apps/tauri/issues/10452
